CHANGELOG.md

```diff
@@ -14,6 +15,10 @@ All notable changes to this project will be documented in this file. The format
 
 ## [Unreleased]
 
+## [1.1.84] - 2024-06-10
```

Comment:
1.1.84 is not up to date anymore. Next version will be v1.1.86

Reply:
Changed to v1.1.87